axelrod/strategies/_strategies.py

```diff
@@ -143,6 +143,7 @@
     OppositeGrudger,
     SoftGrudger,
     SpitefulCC,
+    Capri,
```

Comment:
Can you move this so that the tuple is in alphabetical order please and then `Capri` needs to be added to the `all_strategies` list in this file. (This is why the CI is failing).

Reply:
Fixed.